geotrek/tourism/forms.py

```diff
@@ -101,3 +101,11 @@ def __init__(self, *args, **kwargs):
         # Since we use chosen() in trek_form.html, we don't need the default help text
         for f in ['themes', 'source']:
             self.fields[f].help_text = ''
+
+    def clean(self, *args, **kwargs):
+        data = super().clean(*args, **kwargs)
+
+        if data.get("end_date") and data.get("end_date") < data.get("begin_date"):
+            self.add_error('end_date', _('Start date is after end date'))
+
+        return data
```

Comment:
Please add test in `geotrek/tourism/tests/test_forms.py`
Initialize EventForm with bad data and assert this message is in the list of errors

Examples : https://github.com/GeotrekCE/Geotrek-admin/blob/master/geotrek/core/tests/test_forms.py